sklearn/compose/tests/test_target.py

```diff
@@ -332,3 +332,40 @@ def test_transform_target_regressor_route_pipeline():
     pip.fit(X, y, **{'est__check_input': False})
 
     assert regr.transformer_.fit_counter == 1
+
+
+class DummyRegressorWithExtraPredictParams(DummyRegressor):
+    def predict(self, X, return_std=False, check_input=True):
```

Comment:
Maybe remove `return_std`, since False is the default for `DummyRegressor.predict` anyway.